server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
Hey @tombuildsstuff only have just seen this. My GitHub notifications are all over the place, i'm busy this week :(

Are you around next?